src/routes/github/create-branch/github-create-branch-post.ts

```diff
@@ -5,20 +5,61 @@ import { AnalyticsEventTypes, AnalyticsTrackEventsEnum } from "interfaces/common
 import { statsd } from "~/src/config/statsd";
 import { metricCreateBranch } from "~/src/config/metric-names";
 import { getCloudOrServerFromGitHubAppId } from "~/src/util/get-cloud-or-server";
+import { Subscription } from "models/subscription";
 
-const errorMessages = {
-	// TODO: Fix the url later, once you figure out how to get the `installationId`
-	403: "We couldn’t create this branch, possibly because this GitHub repository hasn't been configured to your Jira site. <a href='#'>Allow access to this repository.</a>",
-	400: "We couldn’t create this branch. Check that you’ve entered valid values for repository, source branch name, and new branch name.",
-	422: "We couldn’t create this branch, possibly because:<ul>" +
-		"<li>This GitHub branch already exists. Please try a different branch name.</li>" +
-		"<li>The GitHub branch name is not valid.</li>" +
-		"</ul>",
-	404: "We couldn’t create this branch, possibly because the GitHub source branch you entered doesn't exist. Please try a different branch.",
-	500: "We couldn’t create this branch because something went wrong. Please try again."
+/**
+ * Returns the URL for Installation settings page in GitHub
+ *
+ * @param userLogin
+ * @param hostname
+ * @param jiraHost
+ * @param repoOwner
+ * @param repoName
+ */
+const getGitHubConfigurationLink = async (
+	userLogin: string,
+	jiraHost: string,
+	hostname: string,
+	repoOwner: string,
+	repoName: string
+) => {
+	// URL for the app configuration for different repoOwners/organizations in GitHub, used during 403 errors
+	let url = `${hostname}/organizations/${repoOwner}/settings/installations/`;
+
+	// If the repoOwners/organizations is the same as the user's login, then the url is different
+	if (userLogin === repoOwner) {
+		url = `${hostname}/settings/installations/`;
+	}
+
+	const subscription = await Subscription.findForRepoNameAndOwner(repoName, repoOwner, jiraHost);
+
+	return url + subscription?.gitHubInstallationId;
+};
+
+const errorMessages = (statusCode: number, url?: string): string => {
```

Comment:
As this is a function, shouldn't the name contains some action? "Get" ? "Map"?

Reply:
Renamed to `getGitHubConfigurationLink`.